extensions/hadoop/src/test/java/com/hazelcast/jet/hadoop/file/CsvFileFormatTest.java

```diff
@@ -127,6 +127,6 @@ public void shouldThrowWhenWrongFormatting() {
                                                     .glob("file-invalid.csv")
                                                     .format(FileFormat.csv(User.class));
 
-        assertJobFailed(source, CsvMappingException.class, "Too many entries");
+        assertJobFailed(source, CsvReadException.class, "Too many entries");
```

Comment:
This change seems to be valid also for master

Reply:
Not sure what you mean. It was part of the original PR to master 
https://github.com/hazelcast/hazelcast/pull/21524/files